.github/workflows/ci.yml

```diff
@@ -59,8 +52,7 @@ jobs:
         # first load any .py files, which are the examples, and then try to run
         # any doctests, there isn't any)
 
-        pytest tests/ src/ examples/ --cov=sklearn_evaluation --doctest-modules --durations-min=5
-
+        pytest tests/ --cov=sklearn_evaluation --doctest-modules --durations-min=5 -n auto        
```

Comment:
removing `src/` will prevent the doctests from running, which are important.

Do they break with the -n auto? if so, we can break them into two pytest calls

are there any learnings from using xdist? we should document this in the [contributing](https://github.com/ploomber/contributing) repo so whoever implements this in other repos can get up to speed 

the interesting thing I'm seeing here is that this is improving performance a lot on windows and mac but ubuntu remains pretty much the same

Reply:
Let me check and push.
Sure I'll open a PR